shared-services/src/files/cloudinary-file.service.ts

```diff
@@ -0,0 +1,21 @@
+import { SetStateAction } from "react"
+import { cloudinaryPdfFromId, cloudinaryUrlFromId } from "@bloom-housing/shared-helpers"
+import { cloudinaryFileUploader } from "../../../sites/partners/src/lib/helpers"
+import { FileServiceInterface } from "./file-service.interface"
+
+export class CloudinaryFileService implements FileServiceInterface {
```

Comment:
I'm new to typescript but I have a question about creating this new interface. Is it practical  to rename this class to something generic? The idea being to keep the class name generic and when we swap out implementations we simply change the function implementation within this class; the advantage is not changing the class names in all references with a different (S3, other CDN, etc.) name.

Reply:
The idea is to have more than one implementation (so CloudinaryFileService, S3FileService, etc.) and then decide which one to use using config, as Chris explained here https://docs.google.com/document/d/17SkUZJGT51TAZuxKZgLHyUmdifrpoborvR8gWVGq_BQ/edit?pli=1#bookmark=id.ij1sdji1xao2. So we still want to have the specific names since the implementation will differ, and even if we use S3 only for Doorway, the idea is for this to be reusable for other future implementations so they can use either/any other service.